www-apps/zeppelin-bin/zeppelin-bin-0.11.2.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit java-pkg-2 verify-sig
+
+MY_PN="zeppelin"
+MY_P="${MY_PN}-${PV}-bin-all"
+
+DESCRIPTION="Web-based interactive data analytics notebook launcher"
+HOMEPAGE="https://zeppelin.apache.org"
+SRC_URI="mirror://apache/zeppelin/${MY_PN}-${PV}/${MY_P}.tgz -> ${P}.tgz
+	verify-sig? ( https://downloads.apache.org/zeppelin/zeppelin-${PV}/${MY_P}.tgz.asc -> ${P}.tgz.asc )"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="Apache-2.0 MIT OFL-1.1 WTFPL-2 BSD BSD-2 CC0-1.0 CDDL EPL-1.0 EPL-2.0"
```

Comment:
Thank you for doing this.

Reply:
All done. Thank you for providing the details.